### PR TITLE
Add details on perplexity to README.md

### PR DESCRIPTION
**Pull request by glinscott**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,6 +240,37 @@ or
...
+You can pass `--perplexity` as a command line option to measure perplexity over the given prompt.  For more background,
+see https://huggingface.co/docs/transformers/perplexity.  However, in general, lower perplexity is better for LLMs.
+
+#### Measurements
+
+https://github.com/ggerganov/llama.cpp/pull/270 is the unofficial tracking page for now.  llama.cpp is measuring very well
+compared to the baseline implementations.  Quantization has a small negative impact to quality, but, as you can see, running
+13B at q4_0 beats the 7B f16 model by a significant amount.
+```
+Perplexity - model options
```
Comment (Green-Sky) [inline README.md:254]: add for which corpus those results are.

**Review by Green-Sky — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (slaren) [reply README.md:254]: Probably should also include the context size used, if it can make a meaningful difference.

Comment (Green-Sky) [reply README.md:254]: while we are at it, make it a table with models vertical - parameters horizontal

Comment (glinscott) [reply README.md:257]: Added dataset details/context length.  For the table, I'm not sure we have enough details across the different models for that to work well right now (will be mostly empty for 13B).

**Review by Green-Sky — APPROVED** [review]

**Pull request merged**